automation/taskcluster/decision_task.py

```diff
@@ -76,16 +76,89 @@ def create_raw_task(name, description, full_command, scopes = []):
     }
 
 
-def create_module_task(module):
-    return create_task(
-        name='Android Components - Module ' + module,
-        description='Building and testing module ' + module,
-        command="-Pcoverage " + " ".join(map(lambda x: module + ":" + x, ['assemble', 'test', 'lint']))  +
-            " && automation/taskcluster/action/upload_coverage_report.sh",
-        scopes = [
-            "secrets:get:project/mobile/android-components/public-tokens"
-        ])
+def create_module_tasks(module):
+    module_overrides = {
+        ":samples-browser": {
+            "assembleOnly/Test": (
+                ["ServoArm", "SystemUniversal"],
+                [
+                    "GeckoBetaAarch64", "GeckoBetaArm", "GeckoBetaX86",
+                    "GeckoNightlyAarch64", "GeckoNightlyArm", "GeckoNightlyX86",
+                    "GeckoReleaseAarch64", "GeckoReleaseArm", "GeckoReleaseX86"
+                ]
```

Comment:
Can we somehow make this dynamic like in the reference browser?

In theory could this be: library project: assemble, app project: assemble variants in parallel

Reply:
Yeah, we could do that. I've elected a static list to have more control over what exactly we do (e.g. the split b/n test and assemble only tasks), but that of course comes at a cost of having to maintain this list. If things were fully dynamic, we either loose granularity, or need to implement support for it via some other means.